samples/stock/demo/inverted-navigator/demo.js

```diff
@@ -0,0 +1,457 @@
+}, {
+    x: 971098070000,
+    y: 1,
+    lbl: 'Geocaching',
+    details: 'First documented placement of Geocaching, an outdoor recreational activity, in which participants use a Global Positioning System (GPS) receiver or mobile device and other navigational techniques to hide and seek containers, took place on May 3, 2000, by Dave Ulmer of Beavercreek, Oregon.'
+}, {
+    x: 1255094870000,
+    y: 1,
+    labelrank: 100,
+    lbl: 'Highcharts',
+    details: 'Highcharts started as Torstein’s humble quest for a simple charting tool for updating his homepage with snow depth measurements from Vikjafjellet, the local mountain where his family keeps a cabin. Frustrated with the common flash plug-ins, and other proprietary solutions at the time, he decided to build a standards-based solution of his own, and share it with the world. It turns out he was doing more than scratching his itch. Since its launch in 2009, Highcharts has remained the secure choice for cutting-edge charting tools in a highly competitive data visualization market. Later in 2011, the first version og Highcharts Stock was released.'
+}, {
+    x: 1476019670000,
+    y: 1,
+    lbl: 'Laser Interferometer Gravitational-Wave Observatory',
+    details: 'The Laser Interferometer Gravitational-Wave Observatory makes the first observation of gravitational waves, fulfilling Einstein\'s prediction.'
+}, {
+    x: 1602250070000,
+    y: 1,
+    lbl: 'Artificial intelligence-assisted coding',
+    details: ''
+}];
+
+Highcharts.stockChart('container', {
+    chart: {
+        inverted: true,
+        zooming: {
+            mouseWheel: {
+                enabled: false
+            }
+        }
+    },
+    title: {
+        text: 'List of stock historic inventions.',
+        align: 'left'
+    },
+    subtitle: {
+        text: 'Source: Wikipedia',
+        align: 'left'
+    },
+    rangeSelector: {
+        buttons: [{
+            type: 'year',
+            count: 30,
+            text: '30y'
+        }, {
+            type: 'year',
+            count: 100,
+            text: '100y'
+        }]
+    },
+    navigator: {
+        opposite: true,
+        series: {
+            type: 'line',
+            marker: {
+                enabled: true
+            },
+            lineWidth: 0
+        },
+        xAxis: {
+            ordinal: false,
+            reversed: false,
+            tickPixelInterval: 50
+        }
+    },
+    xAxis: {
+        ordinal: false,
+        labels: {
+            enabled: false
+        },
+        tickLength: 0,
+        lineWidth: 0,
+        min: Date.UTC(1900, 0, 1),
+        reversed: false
+    },
+    yAxis: {
+        min: 0.75,
+        max: 2,
+        labels: {
+            enabled: false
+        },
+        gridLineWidth: 0
+    },
+    tooltip: {
+        shared: false,
+        useHTML: true,
+        formatter: function () {
+            let tooltipString = '<div>';
```

Comment:
We should rewrite this `formatter` using string formats as per our recommendation to stay JSON compatible.